README.md

```diff
@@ -46,3 +33,25 @@ For help try:
 ```
 $ docker run --rm thethingsindustries/protoc --help
 ```
+
+### To generate language specific code
+
+1. Make sure you have the `model.proto` file present in `${PWD}`
```

Comment:
you have several arguments referring to /model/proto, I suggest explaining the role of each one.